apps/test/unit/craft/craftTest.js

```diff
@@ -17,7 +17,19 @@ describe('Craft', () => {
       level: {},
       skin: {},
     };
+
+    // Craft.init makes an API call, and if that call fails, tries to display
+    // some UI that will fail to render. Avoid that by ensure our API call has
+    // success
+    // A better approach could be to provide a way to give Craft a mocked cc client
+    const server = sinon.fakeServerWithClock.create();
+    server.respondWith(
+      'GET',
+      'http://localhost:8080/connected',
+      [200, { "Content-Type": "application/json" }, '{"foo": "bar"}']);
+
```

Comment:
FYI for @joshlory (and maybe @balderdash ) who I think have spent some time on MC code-connect